templates/web/base/admin/extra-metadata-form.html

```diff
@@ -6,7 +6,13 @@
         <div class="admin-hint"><p>[% loc('The ordering of this field on the report page. Fields are shown in ascending order according to this value.') %]</p></div>
         <label>
             [% loc('Order') %]
-            <input name="metadata[[% loop.index %]].order" data-field-name="order" type=text value="[% meta.order | html %]">
+            <input name="metadata[[% loop.index %]].order" data-field-name="order" type=text value="[% meta.order OR 0 | html %]">
+        </label>
+
+        <div class="admin-hint"><p>[% loc('Whether the field is displayed to the user, in which case it will be blank, otherwise it will be automatically populated. This field is set usually set automatically.') %]</p></div>
+        <label>
+            [% loc('Automated') %]
+            <input name="metadata[[% loop.index %]].automated" data-field-name="automated" type=text value="[% meta.automated | html %]">
```

Comment:
Would this be better as a `<select>` given the limited number of options?

Reply:
Yeah, good point.

Reply:
I see the select has been added, but the `<input>` is still here unless I'm being dense :)